### PR TITLE
Restructured repo and added GitHub Actions.

### PR DESCRIPTION
**Pull request by TomKimber**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPO-5980

### Change description ###

Restructured repo and added GitHub Actions.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+name: cd
+
+on:
+  workflow_dispatch:
+  push:
+    branches: "master"
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: "0 0 * * Tue"
```
Comment (timja) [inline .github/workflows/cd.yml:9]: something I hit recently is that the json schema regex is case sensitive so if you open this in intellij it screams at you, this fixes it:
```suggestion
    - cron: "0 0 * * TUE"
```

File: .hadolint.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+ignored:
```
Comment (timja) [inline .hadolint.yaml:1]: better to ignore in the docker file with a comment next to it if we can?

Comment (TomKimber) [reply .hadolint.yaml]: Implemented exclusions inline.

**Review by timja — APPROVED** [review]

File: distroless-debug/Dockerfile [hunk 3]
```diff
@@ -39,6 +47,10 @@ ONBUILD ARG JVM_ARGS=${DEV_JVM_ARGS:-'-XX:InitialRAMPercentage=30.0 -XX:MaxRAMPe
 ONBUILD ARG JAVA_AGENT_OPTIONS
 ONBUILD ENV JAVA_AGENT_OPTIONS=${JAVA_AGENT_OPTIONS:--javaagent:/opt/app/applicationinsights-agent-${APP_INSIGHTS_AGENT_VERSION}.jar}
 
+
+# hadolint ignore=DL3022
```
Comment (timja) [inline distroless-debug/Dockerfile:51]: is this doing anything?

Comment (TomKimber) [reply distroless-debug/Dockerfile]: No - Removed.

**Pull request merged**